extra-devel/intel-graphics-compiler/autobuild/prepare

```diff
@@ -3,7 +3,7 @@
 #rm -v "$SRCDIR"/IGC/VectorCompiler/spirv-patches-new/*.patch
 
 abinfo "Patching SPRIV-LLVM-Translator using AOSC-patched Intel patches ..."
-PATCHES_ROOT="$(readlink -f "$SRCDIR"/IGC/VectorCompiler/spirv-patches-10/)"
+PATCHES_ROOT="$(readlink -f "$SRCDIR"/IGC/VectorCompiler/spirv-patches-11/)"
```

Comment:
Could probably use `llvm-config` here for version generation?

Reply:
No, Intel does not work this way, the folder name could be `spirv-patches-new` or something.